layouts/default.vue

```diff
@@ -24,6 +36,12 @@ export default {
   computed: {
     isShowBreadcrumb (): boolean {
       return !isLoginRoute()
+    },
+    bannerText (): string | null {
```

Comment:
You don't need to specify `null` here -- any type can be null.

Also, I think the return value will actually be undefined if bannerText is falsey on line 44 -- but don't specify that as a return value either.